packages/workers-shared/utils/helpers.ts

```diff
@@ -1,22 +1,11 @@
 import { getType } from "mime";
 
-/** normalises sep for windows, and encodes each segment */
-export const encodeFilePath = (filePath: string, sep: string) => {
-	const encodedPath = filePath
-		.split(sep)
-		.map((segment) => encodeURIComponent(segment))
-		.join("/");
+/** normalises sep for windows */
+export const normalizeFilePath = (filePath: string, sep: string) => {
+	const encodedPath = filePath.split(sep).join("/");
```

Comment:
Do we need to concern ourselves over Windows drives? E.g. `C:\foo`?
Or is the filePath guaranteed to be relative?

The fact that we always prefix with `/` suggests that is the case?
Would it be worth asserting that is true in case some one starts calling this function with an absolute path in the future?

Reply:
It's only ever used on relative paths currently, but ive added a check in case :)